### DISCUSSION TITLE
No stations connected at all, is this normal? (And a couple more)

### DISCUSSION THREAD

**Discussion by Fermion96** [xspanger3770/GlobalQuake#135] [Q&A]
I downloaded GlobalQuake and wanted to test it out, but no earthquakes are being detected-I set the threshold for 2.5 magnitude worldwide, and compared GlobalQuake with an earthquake alert map I have on my phone to see if any earthquakes were being missed. And I saw that the program wasn't detecting a single one. The app on my phone would keep refreshing to show a 3.3 earthquake here, a 2.7 earthquake there, but none were being shown on GlobalQuake. I think that no stations being connected to any of the networks might be the core problem, as shown in the screenshot. If this really is a bug, could you please fix this?
![zero stations connected](https://github.com/xspanger3770/GlobalQuake/assets/59562641/4cabbf85-d04b-4ca3-8f60-8dc79eb262bc)
Also, GlobalQuake seems to run fine from the command prompt, but double-clicking on the icon just gives me a JNI exception. Is this a bug too, or am I doing something wrong?

I would also like to add a suggestion. Would it be possible for you to implement graphics options where the user can toggle between fault lines shown/not shown, and station locations shown/not shown?

**Comment by xspanger3770** [comment]
Hi!
In order for GlobalQuake to work, you need to first select the stations in your area in the station selection menu that is accessible by pressing the 'Select Stations' button when you launch GlobalQuake. The selection itself is very easy, just drag a rectangle where you want the stations to be selected. Don't forget to apply some small distance filter (20km) so that you don't have stacked stations at one place.

I would recommend launching GlobalQuake from the command line, but double clicking should also work and I can't really tell why it isn't working for you, maybe a Java version thing.

Thanks for your suggestions, the fault lines were already suggested as you can see in the issues, and station toggle can be added.

Have a nice day,
Jakub
